explainers/doc/CREATING_CUSTOM_EXPLAINER_WITH_MLI_BYOR.md

```diff
@@ -42,7 +42,7 @@ When MLI user starts interpretation, model compatible explainers (from the avail
 ### What is the role of recipes?
 BYOR allows Data Scientists to bring their **own recipes** or leverage the existing, **open-source** recipes to explain models. In this way, the expertise of those creating and using the recipes is leveraged to focus on domain-specific functions to build customizations.
 ### Where can you find open-source recipes?
-The curated open-source recipes currently reside [here](https://github.com/h2oai/driverlessai-recipes/tree/rel-1.9.1/explainers/explainers):
+The curated open-source recipes currently reside [explainers](https://github.com/h2oai/driverlessai-recipes/tree/rel-1.9.1/explainers/explainers) section of Driverless AI recipes GitHub repository:
```

Comment:
Do we need it to be rel-1.9.2 (too)?

Reply:
:man_facepalming: good catch! Thank you @arnocandel! I prepared PR https://github.com/h2oai/driverlessai-recipes/pull/145 to fix it (I cannot push to this branch directly and want to get as much as possible out).